src/detect-pangram.js

```diff
@@ -15,7 +15,18 @@
  * @return {boolean}
  */
 function detectPangram(phrase) {
-  // write code here
-}
+  if (phrase.length < 26) {
+    return false;
+  }
+
+  const isPanagram = phrase.match(/([a-z])(?!.*\1)/gi);
+  const length = isPanagram.length;
```

Comment:
You can write this with one line of code
`return isPanagram.length === 26 `

Reply:
unnecessary variable